include/z64ocarina.h

```diff
@@ -26,10 +26,10 @@ typedef enum {
     /* 19 */ OCARINA_SONG_EVAN_PART1,
     /* 20 */ OCARINA_SONG_EVAN_PART2,
     /* 21 */ OCARINA_SONG_ZELDAS_LULLABY,
-    /* 22 */ OCARINA_SONG_SCARECROW,
+    /* 22 */ OCARINA_SONG_SCARECROW_SPAWN,
     /* 23 */ OCARINA_SONG_TERMINA_WALL,
     /* 24 */ OCARINA_SONG_MAX,
-    /* 24 */ OCARINA_SONG_SCARECROW_LONG = OCARINA_SONG_MAX
+    /* 24 */ OCARINA_SONG_SCARECROW_LONG = OCARINA_SONG_MAX // anything larger than 13 is considered the long scarecrow's song
```

Comment:
Copy-Pasta from OoT for the 13, will update! 

This whole system of `OCARINA_SONG_MAX` and `OCARINA_SONG_SCARECROW_LONG` actually comes from the message system and not the AudioOcarina docs. @Thar0 was the one who did the docs for this in OoT's z_message and this was simply copied over through that. Yes, I believe it might be worth changing, but it might be better to doc this through the message system instead of the ocarina system as this is where this distinction is important